### DISCUSSION TITLE
Limit parallel Executions

### DISCUSSION THREAD

**Discussion by marcohald** [ArchiveBox/ArchiveBox#982] [Q&A]
I'm running ArchiveBox on a small VM with limited resources.
Is it possible to limit the Download to for example 2 Pages at a time ?

**Comment by pirate** [comment]
It only downloads one page at a time unless you explicitly run multiple `archivebox add` commands in separate tabs.

**Reply by marcohald** [reply]
I totally forgot about this question, sorry for that.
My use case would be using a Bookmark with this Link `javascript:void(window.open('http://archivebox/add/?url='+document.location.href));` to trigger the add function  
So this triggers parallel downloads
